content/libraries/multiformats/_index.md

```diff
@@ -1,22 +1,30 @@
 ---
 title: Multiformats
-description: Multiformats - self describing protocol values
-weight: 6
+weight: 4
 dashboardWeight: 1
-dashboardState: missing
-dashboardAudit: n/a
+dashboardState: stable
+dashboardAudit: missing
 dashboardTests: 0
 ---
 
-# Multiformats
+# Multiformats - self describing protocol values
 
-Self-describing protocol values
+[Multiformats](https://multiformats.io/) is a set of formating rules and formatting structures for self-describing values. These values are useful both to the data layer (IPLD) and to the network layer (libp2p). Multiformats includes specifications for the Content Identifier (CID) used by IPLD and IPFS, the multicodec, multibase and multiaddress (used by libp2p).
```

Comment:
I would cut the first sentence down to “Multiformats is a set of self-describing values”